explorer/explorerroutes.go

```diff
@@ -180,6 +180,13 @@ func (exp *explorerUI) TxPage(w http.ResponseWriter, r *http.Request) {
 					tx.TicketInfo.PoolStatus = poolStatus.String()
 				}
 				tx.TicketInfo.SpendStatus = spendStatus.String()
+				var averageWinTimeInBlocks int64 = 8192
```

Comment:
I think this depends on the network. Pool size is different on testnet for example

Reply:
@McEdward https://github.com/decred/dcrd/blob/master/chaincfg/params.go

Reply:
will rectify